cleverpush/src/main/java/com/cleverpush/responsehandlers/ChannelConfigFromBundleIdResponseHandler.java

```diff
@@ -33,7 +33,7 @@ public void onSuccess(String response) {
           JSONObject responseJson = new JSONObject(response);
           cleverPush.setChannelConfig(responseJson);
           cleverPush.setChannelId(responseJson.getString("channelId"));
-          cleverPush.subscribeOrSync(autoRegister);
+          cleverPush.subscribeOrSync(autoRegister, false);
```

Comment:
Instead of providing `false` here please add another method subscribeOrSync(autoRegister) with only one argument which internally calls subscribeOrSync(autoRegister, false); in the main class